src/parser.rs

```diff
@@ -684,17 +684,36 @@ where
         }
         checkpoint
     }
+    fn handle_operation_right(
+        &mut self,
+        once: bool,
+        next: fn(&mut Self) -> Checkpoint,
+        ops: &[SyntaxKind],
+    ) -> Checkpoint {
+        let checkpoint = next(self);
+        if self.peek().map(|t| ops.contains(&t)).unwrap_or(false) {
+            self.start_node_at(checkpoint, NODE_BIN_OP);
+            self.bump();
+            if once {
+                next(self);
```

Comment:
Not sure if I'm missing something, but isn't `once` always `false` here?

Reply:
you are right, let me refactor to remove those. Unfortunately, I simply broke down @oberblastmeister PR without fully understanding the code :sweat_smile: